### PR TITLE
add new border radius tokens

### PR DESCRIPTION
**Pull request by clxandstuff**

### CODE REVIEW

**Review by coderitual — CHANGES_REQUESTED** [review]
hey @clxandstuff , I think that we're missing mdx docs for those tokens in this PR

File: src/components/dialog/_dialog.scss [hunk 1]
```diff
@@ -176,11 +176,11 @@ $dialogMaxWidthMap: (
 
     &--appearance-dialog {
       background-color: $white;
-      border-radius: 24px 24px 0 0;
+      border-radius: var(--border-radius-l) var(--border-radius-l) 0 0;
       box-shadow: $dialogBoxShadow;
 
       @include sgResponsive(md) {
-        border-radius: $borderRadiusDefault;
+        border-radius: var(--border-radius-xs);
```
Comment (lewandowskid) [inline src/components/dialog/_dialog.scss:183]: according to Wojtek's documentation border radius for desktop dialog should be `border-radius: var(--border-radius-l);`

https://www.figma.com/file/hJG4ROfLYw6lymxeVuQRyy/%5BDS%5D-Border-Radius?node-id=228-14520&t=rHnBbnCpt0d0GWJY-4

Comment (clxandstuff) [reply src/components/dialog/_dialog.scss]: done

**Comment by clxandstuff** [comment]
@coderitual
Docs added

**Review by lewandowskid — APPROVED** [review]

**Review by coderitual — APPROVED** [review]

**Pull request merged**